decidim-module-census_connector/app/models/census/api/census_api.rb

```diff
@@ -33,9 +33,15 @@ def send_request
         response = yield
 
         http_response_code = response.status
-        return { http_response_code: http_response_code } if [500, 204].include?(http_response_code)
+        http_response_body = response.body
 
-        json_response = JSON.parse(response.body, symbolize_names: true)
+        if [500, 204].include?(http_response_code)
+          Rails.logger.warn http_response_body
```

Comment:
Would this log have sensible data? We must check that...

Reply:
I don't think so, do you have something in mind?

Reply:
I mean sensible data from users, for example all in the registration form. The idea is to have all sensible data stored on Census servers and nothing in Decidim, even logs.

Reply:
I expect this to include a technical error, maybe the code around a crash and a backtrace. A 500 should always be unexpected, I just logged it because it might be easier to troubleshoot but it's not really necessary since the same information will be available in census anyways.

However, I'm not sure I understand the privacy issue. Are user params not logged anyways before they are sent to census?

Reply:
Ah, if it only logs error responses it's ok.

Reply:
Yeah. Well, also 204 currently but that should have an empty body so it's a bit of a dummy code path...

Reply:
If this is a concern though, we should review crowdfundings, it's full of API response logging.